aspnetcore/toc.md

```diff
@@ -29,6 +29,7 @@
 ## [Open Web Interface for .NET (OWIN)](fundamentals/owin.md)
 ## [Choose between ASP.NET Core and ASP.NET](choose-aspnet-framework.md)
 ## [Choose between .NET Core and .NET Framework runtime](https://docs.microsoft.com/dotnet/articles/standard/choosing-core-framework-server)
+## [Microsoft.AspNetCore.All metapackage](xref:fundamentals/metapackage)
```

Comment:
@tdykstra  my feelings exactly - but this email from Dan.Roth caused me to bury it

>I don’t think the metapackage needs its own doc – it’s just the package you install to use ASP.NET Core. We do need to address the runtime store and publish profiles, but Damian is really the guy to talk to on how that should be documented.